docs/development/extensions-core/lookups-cached-global.md

```diff
@@ -368,6 +368,12 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
 }
 ```
 
+> If using JDBC, you will need to add your database's client JAR files to the extension's directory.
+> For MySQL, you can get it from https://dev.mysql.com/downloads/connector/j/, and for Postgres, from
+> https://jdbc.postgresql.org/download.html or from the `extensions/postgresql-metadata-storage/`.
```

Comment:
Typo: from the `extensions/postgresql-metadata-storage/` -> the `extensions/postgresql-metadata-storage/` directory.

Or: from the `extensions/postgresql-metadata-storage/` -> from `extensions/postgresql-metadata-storage/`

Reply:
Fixed